include/openmc/tallies/filter_material.h

```diff
@@ -46,7 +46,7 @@ class MaterialFilter : public Filter {
 
   void set_materials(gsl::span<const int32_t> materials);
 
-private:
+protected:
```

Comment:
Why was this change necessary?

Reply:
I need to use both `materials_` and `map_` that were private member of the `FilterMaterial` class, those change match what was done in the `FilterCell` class.

Reply:
Ah right, needed so they can be accessed in the inherited class. Thanks!